chalice/cli/__init__.py

```diff
@@ -182,6 +189,59 @@ def deploy(ctx, autogen_policy, profile, api_gateway_stage, stage,
     reporter.display_report(deployed_values)
 
 
+@cli.command('invoke')
+@click.option('-n', '--name', metavar='NAME', required=True,
+              help=('The name of the function to invoke. '
+                    'This is the logical name of the function. The only '
+                    'exception is to invoke one of the routes the name '
+                    'api_handler should be used.'))
+@click.option('--profile', metavar='PROFILE',
+              help='Override profile at deploy time.')
+@click.option('--stage', metavar='STAGE', default=DEFAULT_STAGE_NAME,
+              help=('Name of the Chalice stage to deploy to. '
```

Comment:
Probably should reword the `--stage` help command here. Looks like this is from the `deploy` command.